app/presenters/summary/sections/income_benefits_details.rb

```diff
@@ -0,0 +1,70 @@
+module Summary
+  module Sections
+    class IncomeBenefitsDetails < Sections::BaseSection
+      def show?
+        income.present? && super
+      end
+
+      def answers # rubocop:disable Metrics/AbcSize, Metrics/MethodLength, Metrics/PerceivedComplexity, Metrics/CyclomaticComplexity
+        if income_benefits.empty?
+          [
+            Components::ValueAnswer.new(
+              :which_benefits, 'none',
+              change_path: edit_steps_income_income_benefits_path
+            )
+          ].flatten.select(&:show?)
+        else
+          absent_payment_types = IncomeBenefitType.values.filter_map do |type|
+            type.value.to_s unless type.value == :none
+          end
+          reported_payment_types = income_benefits.map(&:payment_type)
+          absent_payment_types -= reported_payment_types
```

Comment:
I interpreted the figma designs to display the payments in that order rather than all the absent payment types at the bottom. I think that's how i implemented it in review